src/vs/workbench/browser/quickaccess.ts

```diff
@@ -45,3 +47,54 @@ export function getQuickNavigateHandler(id: string, next?: boolean): ICommandHan
 		quickInputService.navigate(!!next, quickNavigate);
 	};
 }
+
+export function getSelectionTextFromEditor(allowUnselectedWord: boolean, activeEditor: IEditor): string | '' {
```

Comment:
```suggestion
export function getSelectionTextFromEditor(allowUnselectedWord: boolean, activeEditor: IEditor): string {
```

It's obvious from the body that empty string is returned

Reply:
Done!